src/util/piped_process.h

```diff
@@ -0,0 +1,39 @@
+#ifndef CPROVER_UTIL_PIPED_PROCESS_H
+#define CPROVER_UTIL_PIPED_PROCESS_H
+
+// TODO: Revisit buffer size and storage location.
+// Note that this may impact number of characters correctly communicated
+// by the pipe, to check/fix.
```

Comment:
It shouldn't affect correctness, if it does, yeah, please reconsider!

Reply:
It doesn't, we have verified it.